### ISSUE TITLE
Some fp16 problems when extending solofusion to the large backbone.

### ISSUE THREAD

**Issue by WeijieMax** [Divadi/SOLOFusion#12]
Dear author,

When I extend SOLOFusion to ConvNext-B, it can't converge well according to your provided config settings. However, when I canceled the fp16 hook and re-trained it, the loss tendency was normal. But there is no doubt that the re-implementation performance will differ from yours. Is there any revision when training the model of the large version?

**Comment by Divadi** [comment]
Hello - I'm not entirely sure what settings you used to train the ConvNeXt-B model. I loaded weights from mmclassification's convnext and and adopted some training settings from PETR and mmdetection's adaptation of the weights.

Specifically with regards to FP16, I am assuming that you also increased the BEV resolution? This causes the heatmap loss to grow as well, potentially going beyond 65536 for initial iterations. So, I set warmup_loss_scale_value to be 0.1 instead of 1.

Please let me know if you have any other questions.
